requirements/pytorch/loggers.txt

```diff
@@ -1,8 +1,5 @@
-# NOTE: the upper bound for the package version is only set for CI stability, and it is dropped while installing this package
-#  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
-
-# all supported loggers
-neptune-client>=0.10.0, <0.16.10
-comet-ml>=3.1.12, <3.31.16
-mlflow>=1.0.0, <1.29.0
-wandb>=0.10.22, <0.13.4
+# all supported loggers. we don't install them in CI but are here as a reference of the supported loggers
```

Comment:
I've added back this file for the doc's mocks and just as a brief reference of which loggers we support, but I don't think it makes sense to expose this as `pytorch_lightning[loggers]` if we don't use it since there's no reason for users to install all the loggers, normally the'd just install the one they use